tests/requirements.txt

```diff
@@ -0,0 +1,6 @@
+pytest~=8.3
+jinja2~=3.1
+pyopenssl~=24.3
+niquests~=3.11
+grpcio~=1.68
+opentelemetry-proto~=1.29
```

Comment:
We shouldn't bump these versions, unless we really require newer one.

Reply:
I locally tested it before changing it. While the merge isn't complete, I think it's okay.

Reply:
Do we really require 1.29.x? I'd assume - no and thus we shouldn't dump this version.

Reply:
We don't require 1.28.x, either. I do not see any issues here. The same is true for the otelcol version. We don't actually require the last version to test, but it's nice to have. I have a concern when we use a version that is not locally checked.

Reply:
Didn't we discuss this already? The whole point of using `~=` is to stop manually updating these versions.